HIPs/firefly.md

```diff
@@ -0,0 +1,379 @@
+---
+layout: default
+title: HIP Template
+parent: Hyperledger Firefly
+---
+
+# Hyperledger FireFly Proposal
+
+[https://github.com/kaleido-io/hyperledger-hip/blob/gh-pages/HIPs/0000-template.md](https://github.com/kaleido-io/hyperledger-hip/blob/gh-pages/HIPs/0000-template.md)
```

Comment:
This should change to reflect this document. Maybe:
`[HIPs/firefly.md](./firefly.md)`

Reply:
updated as suggested